app/(admin)/layout.tsx

```diff
@@ -1,12 +1,23 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
-import React, { JSX } from 'react';
-import { GeistSans } from 'geist/font/sans';
 import '../globals.css';
 import { AuthContextProvider } from '@/context/AuthContextProvider';
-import ErrorBoundary from '../error';
+import { GeistSans } from 'geist/font/sans';
+import {
+  LucideBell,
+  LucideChevronsUpDown,
+  LucideCog,
+  LucideHome,
+  LucideLayoutGrid,
+  LucideUsers,
+} from 'lucide-react';
 import { Toaster } from 'react-hot-toast';
+import ErrorBoundary from '../error';
+import Heading from '@/components/Heading/Heading';
+import Link from 'next/link';
```

Comment:
Should LinkCustom be used instead?

Reply:
This will be removed when I componetize things. So right now it's a non-issue.

Reply:
@vmaineng I'm not using the `LinkCustom` component here (well, in the [nav component](https://github.com/LetsGetTechnical/gridiron-survivor/blob/ryan/create-admin-layout/components/AdminNav/AdminNav.tsx)) because these links have different styles than others as per the design.
![Figma Gridiron Survivor App-000026](https://github.com/user-attachments/assets/18009f57-d4b5-4787-a690-8ec564529a69)